docs/index.bs

```diff
@@ -3175,6 +3222,52 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Return |response|.
   </section>
 
+  <section algorithm>
+    <h3 id="parse-urlpattern-algorithm"><dfn>ParseURLPattern</dfn></h3>
+      : Input
+      :: |rawPattern|, a {{RouterCondition/urlPattern}}"
+      :: |serviceWorker|, a [=/service worker=]
+      : Output
+      :: {{URLPattern}}
+
+      1. Let |baseURL| be |serviceWorker|'s [=service worker/script url=].
+      1. Return the result of [=urlpattern/building a URLPattern from a WebIDL value=] |rawPattern| given |baseURL| and |serviceWorker|'s [=service worker/global object=]'s [=relevant realm=].
```

Comment:
```suggestion
      1. Return the result of [=building a URLPattern from a WebIDL value=] |rawPattern| given |baseURL| and |serviceWorker|'s [=service worker/global object=]'s [=relevant realm=].
```

This change should probably allow you to remove line 139.

However, after https://github.com/whatwg/urlpattern/pull/201 is merged, you will need to update accordingly, from `WebIDL` to `Web IDL`.